src/briefcase/integrations/cookiecutter.py

```diff
@@ -77,6 +77,28 @@ def escape_non_ascii(obj):
         environment.filters["escape_non_ascii"] = escape_non_ascii
 
 
+class GradleEscape(Extension):
+    """Jinja2 extension to escape strings for Gradle as well."""
+
+    def  __init__(self, environment):
+        """Initialize the extension with the given environment."""
+        super().__init__(environment)
+
+        def escape_toml(obj):
+            """Escapes single quotes."""
+            return obj.replace("'", '\\"')
+
+        # def escape_non_ascii(obj):
+        #     """Quotes obj if non ascii characters are present."""
+        #     if obj.isascii():
+        #         return obj
+        #     else:
+        #         return '"' + obj + '"'
+
```

Comment:
If Gradle doesn't require non-ascii escaping, we can delete this block rather than commenting it out.